npm/webpack-dev-server/package.json

```diff
@@ -16,6 +17,10 @@
   },
   "devDependencies": {
     "@types/webpack-dev-server": "^3.11.1",
+    "chai": "^4.2.0",
+    "mocha": "^8.1.3",
```

Comment:
do we need both mocha and TS mocha ? 🤔

Reply:
Yep, ts-mocha has mocha as a peer dep - better to lock the version.